Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -171,28 +163,24 @@ class TabLocationView: UIView {
     
     lazy var pageOptionsButton: ToolbarButton = {
         let pageOptionsButton = ToolbarButton(frame: .zero)
-        pageOptionsButton.setImage(UIImage.templateImageNamed("menu-More-Options"), for: .normal)
+        pageOptionsButton.setImage(#imageLiteral(resourceName: "menu-More-Options").template, for: .normal)
         pageOptionsButton.addTarget(self, action: #selector(didPressPageOptionsButton), for: .touchUpInside)
         pageOptionsButton.isAccessibilityElement = true
         pageOptionsButton.isHidden = true
         pageOptionsButton.imageView?.contentMode = .left
         pageOptionsButton.accessibilityLabel = NSLocalizedString("Page Options Menu", comment: "Accessibility label for the Page Options menu button")
         pageOptionsButton.accessibilityIdentifier = "TabLocationView.pageOptionsButton"
-        let longPressGesture = UILongPressGestureRecognizer(target: self, action: #selector(didLongPressPageOptionsButton))
-        pageOptionsButton.addGestureRecognizer(longPressGesture)
         return pageOptionsButton
     }()
     
-    lazy var shieldsButton: ToolbarButton = {
-        let button = ToolbarButton()
-        button.setImage(UIImage(imageLiteralResourceName: "shields-menu-icon"), for: .normal)
-        button.addTarget(self, action: #selector(tappedBraveShieldsButton), for: .touchUpInside)
-        button.isAccessibilityElement = true
-        button.imageView?.contentMode = .left
-        button.accessibilityLabel = Strings.Brave_Panel
-        button.accessibilityIdentifier = "TabLocationView.shieldsButton"
-        return button
-    }()
+    lazy var reloadButton = ToolbarButton().then {
+        $0.accessibilityIdentifier = "TabToolbar.stopReloadButton"
+        $0.accessibilityLabel = NSLocalizedString("Reload", comment: "Accessibility Label for the tab toolbar Reload button")
+        $0.setImage(#imageLiteral(resourceName: "nav-refresh").withRenderingMode(.alwaysTemplate), for: .normal)
+        let longPressGestureStopReloadButton = UILongPressGestureRecognizer(target: self, action: #selector(didLongPressStopReload(_:)))
```

Comment:
I don't like naming selectors like this `didLongPressStopReload`, is there any naming convention for selectors or should we decide on something? I usually use normal function names, don't really follow any special naming for selector targets.
I like those `did..` `will..` on delegates only.

Reply:
I usually name these things without the `did` part. i.e. `tappedSomeButton`, `longPressedRefreshButton`, etc.  Only named like this for consistency/ported what it was already named I believe.

We can definitely decide something, perhaps we can add it to the doc for friday if its not already there